source/includes/connection-snippets/x509-snippets/c-connection.c

```diff
@@ -0,0 +1,30 @@
+// begin x509 connection
+#include <mongoc/mongoc.h>
+
+int
+main (int argc, char *argv[])
+{
+   mongoc_database_t *database;
+   mongoc_client_t *client;
+
+   mongoc_init ();
+
+   const mongoc_ssl_opt_t *ssl_default = mongoc_ssl_opt_get_default ();
+   mongoc_ssl_opt_t ssl_opts = { 0 };
+
+   memcpy(&ssl_opts, ssl_default, sizeof ssl_opts);
```

Comment:
Per earlier comments about coding style (https://github.com/mongodb/docs-ecosystem/pull/573/files#r347552954), you may want a space between `memcpy` and the opening paren here.